spark/v3.2/build.gradle

```diff
@@ -31,10 +30,10 @@ configure(sparkProjects) {
   configurations {
     all {
       resolutionStrategy {
```

Comment:
Not related to this PR, but wonder why the `resolutionStrategy` for 3.1/3.2 differs from 3.3/3.4.

Reply:
You mean the why did `force libs.jackson211.module.paranamer` disappear from resolutionStrategy in 3.3 & 3.4?  Not sure, I checked a dependency tree using gradle's dependencies command and paranamer dependency is present in a both 3.4 and 3.2, but paranamer is drooped from resolutionStrategy for 3.4 module.